### DISCUSSION TITLE
etcd cluster cannot join new node

### DISCUSSION THREAD

**Discussion by seqwait** [emqx/emqx#5197] [Q&A]
1.  version: 4.0
2. docker 19.03.8
cluster:
Cluster status: #{running_nodes =>
                      ['emqx1@172.17.110.196','emqx2@172.17.110.203',
                       'emqx3@172.17.110.204'],
                  stopped_nodes => []}

**Comment by k32** [comment]
Hello,

Can you check network connectivity between the nodes and setup the firewall openings according to the instruction: https://docs.emqx.io/en/broker/v4.3/advanced/cluster.html#firewall-settings
Please note that if you name nodes `emqx1@...`, `emqx2@....`, the numeric suffix will affect the ports that should be opened.
